src/nvim/terminal.c

```diff
@@ -644,6 +653,57 @@ Buffer terminal_buf(const Terminal *term)
 // }}}
 // libvterm callbacks {{{
 
+#define STATIC_CSTR_AS_STRING(s) ((String) {.data = s, .size = sizeof(s) - 1})
```

Comment:
this is already in `api/private/helpers.h`. Why's it needed here?

Reply:
I removed it anyway in the latest version, because a) it'd need `strlen()` instead of `sizeof` and b) I can pass a different size (-1 for the newline).